src/context/mod.rs

```diff
@@ -251,15 +251,29 @@ pub trait Context<'a>: ContextInternal<'a> {
     }
 
     /// Convenience method for creating a `JsNull` value.
+    #[cfg(feature = "legacy-runtime")]
```

Comment:
Since these methods have identical function signatures, we also have the option of putting the feature flag inside the method. I'm undecided on which is cleaner. Thoughts?

_Note: This comment is applicable to multiple lines._

Reply:
Yeah, I was looking at that too, but AFAIK you have to put things in a `{ block }` to put an attribute on them:
```rust
fn null(&mut self) -> Handle<'a, JsNull>
	#[cfg(feature = "legacy-runtime")]
    {
        JsNull::new()
    }
	#[cfg(feature = "napi-runtime")]
    {
        JsNull::new(self)
    }
}
```
and that just looked ugly :upside_down_face: 

Saying that, while it looks ugly, it might be actually be clearer.

Reply:
Ok I think it's better if we don't have to manually check if the signature is identical when working with these methods in the future, so I moved the attributes inside the function. With an explicit `return` we can make it quite nice I think:
```rust
#[cfg(feature = "legacy-runtime")]
return JsNull::new();
#[cfg(feature = "napi-runtime")]
return JsNull::new(self);
```